src/enveasy/__init__.py

```diff
@@ -0,0 +1 @@
+from enveasy.cli import app
```

Comment:
No Need , we can remove it

Reply:
> No Need , we can remove it

@Mr-Sunglasses if this is removed, how will enveasy will reference to it, when used via terminal?